package.json

```diff
@@ -38,10 +38,12 @@
     "lodash": "4.17.4",
     "node-fetch": "1.7.1",
     "read": "1.0.7",
+    "semver": "5.3.0",
     "string-length": "1.0.1",
     "through2": "2.0.3",
     "tmp": "0.0.31",
-    "update-notifier": "2.2.0"
+    "update-notifier": "2.2.0",
+    "yamljs": "0.3.0"
```

Comment:
Wait a minute... Those 2 packages are required in running the script and not in developing it.